doc/rest-api/Metrics-backend.md

```diff
@@ -35,24 +35,25 @@ The above configuration starts the API only on a loopback interface.
 The responses are composed in a JSON format with a root element containing one or more attributes as response elements.
 
 Example response:
-
-    {
-        "hosts": [
-            "localhost"
-        ],
-        "metrics": [
-            "xmppErrorIq",
-            "xmppPresenceReceived",
-            "xmppMessageBounced",
-            (...)
-        ],
-        "global": [
-            "nodeSessionCount",
-            "totalSessionCount",
-            "uniqueSessionCount",
-            (...)
-        ]
-    }
+```json
+{
+    "hosts": [
+        "localhost"
+    ],
+    "metrics": [
+        "xmppErrorIq",
+        "xmppPresenceReceived",
+        "xmppMessageBounced",
+        (...)
```

Comment:
Can we mark these `(...)` as a JSON comment? So that github doesn't show it all red like it does here?